converters/toZigbee.js

```diff
@@ -3723,6 +3723,20 @@ const converters = {
             const scenename = '';
             const transtime = value.hasOwnProperty('transition') ? value.transition : 0;
 
+            // Set correct meta.mapped for groups
+            // * all definition metas are the same -> copy meta.mapped[0]
+            // * mixed device models -> meta.mapped = null
+            if (isGroup && entity.members.length > 0) {
```

Comment:
This code is a dupe of https://github.com/Koenkk/zigbee-herdsman-converters/blob/f8226da029df72fa82c3514284bfcc553dec6741/converters/toZigbee.js#L762, can you create a function out of this in utils.js?

Reply:
Indeed it is, seems reasonable to abstract that. I'll try to do it tomorrow.

Reply:
Still needs testing, but can't move the PR back to draft it seems